src/components/gallery/ImageGrid.jsx

```diff
@@ -0,0 +1,100 @@
+import {
+  IonCol,
+  IonGrid,
+  IonInfiniteScroll,
+  IonInfiniteScrollContent,
+  IonRow,
+} from "@ionic/react";
+import { image } from "ionicons/icons";
+import { useEffect, useState } from "react";
+import { getItemsByCollectionId } from "../../services/items";
+import FlexImage from "../image/FlexImage"
+import "./styles.scss";
```

Comment:
Should we standardise the naming convention of `.scss`? i.e. `Component.scss` or `style.scss`